### PR TITLE
chore: prepare 3.4.0 release

### PR DESCRIPTION
**Pull request by grignards**
See #3907 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md` and notify relevant people of this change

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Review by raphj — APPROVED** [review]

File: changelog.md [hunk 1]
```diff
@@ -1,9 +1,18 @@
+## 3.4.0 / 2020-12-21
 
+### New Features
+
+- Files uploaded to Tracim can now be stored on a Amazon S3 compatible server, see [backend documentetation](backend/doc/setting.md) for details on how to configure it.
```
Comment (inkhey) [inline changelog.md]:
```suggestion
- Files uploaded to Tracim can now be stored on a Amazon S3 compatible server, see [backend documentation](backend/doc/setting.md) for details on how to configure it.
```
maybe but some warning about the fact the feature is in some kind of "beta" stage ? (as preview are not efficiently generated with it for now ?)

**Review by inkhey — CHANGES_REQUESTED** [review]

**Review by inkhey — CHANGES_REQUESTED** [review]
We can also have a simple text about experimental gocryptfs encrypted docker.

**Comment by grignards** [comment]
> We can also have a simple text about experimental gocryptfs encrypted docker.

I'm not sure that documenting it in the changelog is a good thing: we do not plan to keep it for long, I do not see the need to create visibility for it.

File: changelog.md (hunk 1 above)
Comment (grignards) [reply changelog.md]: Added a sentence to explain this.

**Review by inkhey — APPROVED** [review]

**Pull request merged**